javascript/modules/client/src/client-addressList.ts

```diff
@@ -397,8 +397,8 @@ export class ClientAddressList extends ClientCore
     try {
       await this.graphql.ensureOnline();
       const client = this.graphql.getClient();
-      const { whitelistProposal: addressListProposal }: {
-        whitelistProposal: SubgraphAddressListProposal;
+      const { allowlistProposal: addressListProposal }: {
```

Comment:
address list?

Reply:
nope, this is the object of subgrpah, which still has the allowlist naming